docs/communication.rst

```diff
@@ -0,0 +1,31 @@
+.. _communication:
+
+*************
+Communication
+*************
+
+We welcome your feedback, questions and ideas. Here's how to reach the community.
+
+Code of Conduct
+===============
+
+All communication and interactions in the Ansible Builder community are governed by the `Ansible code of conduct <https://docs.ansible.com/ansible/devel/community/code_of_conduct.html>`_. Please read and abide by it!
+Reach out to our community team at `codeofconduct@ansible.com <mailto:codeofconduct@ansible.com>`_ if you have any questions or need assistance.
+
+Forum
+=====
+
+Join the `Ansible Forum <https://forum.ansible.com>`_ as a single starting point and our default communication platform for questions and help, development discussions, events, and much more. `Register <https://forum.ansible.com/signup?>`_ to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
```

Comment:
```suggestion
Join the `Ansible Forum <https://forum.ansible.com>`_, the default communication platform for questions and help, development discussions, events, and much more. `Register <https://forum.ansible.com/signup?>`_ to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
```

Reply:
I think "single starting point" makes sense to us but maybe not so much to arbitrary folks out there in the wild.